src/main/java/frc/robot/commands/arm/JiggleNote.java

```diff
@@ -23,21 +23,22 @@ public void initialize() {
 
     @Override
     public void execute() {
+
+    }
```

Comment:
I think you can delete this? the superclass should provide a default implementation which similarly does nothing